THANKS.txt

```diff
@@ -130,7 +130,9 @@ Jörg Dietrich for the k-sample Anderson Darling test.
 Blake Griffith for improvements to scipy.sparse.
 Andrew Nelson for scipy.optimize.differential_evolution.
 Brian Newsom for work on ctypes multivariate integration.
-Nathan Woods for advising on multivariate integration and building unit tests.
+Nathan Woods for work on multivariate integration. His work from 31 Aug 2015
+    to the present is subject to the disclaimer at:
+    https://github.com/losalamos/Gazebo/blob/master/LICENSE.
```

Comment:
Hmm, such a link doesn't seem very future-proof. And it's basically the same license as SciPy uses (3-clause BSD) except for the sentence where it mentions the contract number under which it is produced. It would be simpler  to just say that your contributions are copyright LANL.